### ISSUE TITLE
Add the `no-consecutive-br` rule

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#1253]
Want the rule to warn about consecutive `<br>` tags.

```html
<!-- ❌ incorrect -->
<p>Lorem ipsum dolor sit amet<br>
<br>
consectetur adipisicing elit</p>
```

```html
<!-- ✅ correct -->
<p>Lorem ipsum dolor sit amet</p>
<p>consectetur adipisicing elit</p>
```
